### PR TITLE
DOC-6204: Added release notes for  4.9, 4.10 and 4.11 patches

### PR DESCRIPTION
**Pull request by AnnaK-R3**
Added release notes for the 4.9, 4.10, and 4.11 patches.
Preview: https://neil.preview.docs.r3.com/en/platform/corda/4.11/enterprise/release-notes-enterprise.html

### CODE REVIEW

**Review by adelel1 — APPROVED** [review]

File: content/en/platform/corda/4.10/community/release-notes.md [hunk 1]
```diff
@@ -20,6 +20,19 @@ tags:
...
 # Corda Community Edition 4.10 release notes
 
+## Corda Community Edition 4.10.4 release notes
+
+Corda Community Edition 4.10.4 is a patch release of Corda Community Edition focused on resolving issues.
+
+### Fixed issues
+
+* In the default log4j2.xml file, the Delete action in the DefaultRolloverStrategy policy for log files beginning with `diagnostic-*` or `checkpoints_agent-*`  was incorrect. It erroneously compared against the wrong file names. This issue has been rectified, ensuring that files are now deleted in accordance with the policy.
+* Previously, a rare error scenario could occur where a node would erroneously perceive a valid connection to a peer when, in fact, it was not connected. This issue typically arose when the peer node was disconnecting/connecting. This issue has now been resolved.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.10/community/release-notes.md:30]:
```suggestion
* Previously, a rare error scenario could occur where a node would erroneously perceive a valid connection to a peer when, in fact, it was not connected. This issue typically arose when the peer node was disconnecting/connecting.

File: content/en/platform/corda/4.10/enterprise/release-checksum-enterprise.md [hunk 2]
```diff
@@ -20,6 +20,28 @@ The Corda Enterprise release consists of multiple release packs. You can downloa
...
 
+## Release packs and checksums for Corda Enterprise Edition 4.10.4
+
+The release packs and checksum files for Corda Enterprise 4.10.4 can be downloaded from the Customer Hub.
+
+The following release packs are available, along with their respective checksum files:
+
+* devtest-4.10.4.tar.gz
+
+  Required for building, developing and testing CorDapps.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.10/enterprise/release-checksum-enterprise.md:31]:
```suggestion
  Required for building, developing, and testing CorDapps.
```

File: content/en/platform/corda/4.10/enterprise/release-notes-enterprise.md [hunk 3]
```diff
@@ -17,6 +17,29 @@ weight: 10
...
+
+Corda Enterprise Edition 4.10.4 is a patch release of Corda Enterprise Edition focused on resolving issues.
+
+### Upgrade recommendation
+
+As a developer or node operator, you should upgrade to the [latest released version of Corda]({{< relref "../enterprise/_index.md" >}}) as soon as possible. The latest Corda Enterprise release notes are on this page, and for the latest upgrade guide, refer to [Upgrading a CorDapp or node](upgrading-index.md).
+
+### Fixed issues
+
+* In the default log4j2.xml file, the Delete action in the DefaultRolloverStrategy policy for log files beginning with `diagnostic-*` or `checkpoints_agent-*`  was incorrect. It erroneously compared against the wrong file names. This issue has been rectified, ensuring that files are now deleted in accordance with the policy.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.10/enterprise/release-notes-enterprise.md:34]:
```suggestion
* In the default log4j2.xml file, the Delete action in the DefaultRolloverStrategy policy for log files beginning with `diagnostic-*` or `checkpoints_agent-*` was incorrect. It erroneously compared against the wrong file names. This issue has been rectified, ensuring that files are now deleted in accordance with the policy.
```

File: content/en/platform/corda/4.10/enterprise/release-notes-enterprise.md [hunk 4]
```diff
@@ -17,6 +17,29 @@ weight: 10
...
+* Resolved a TLS connection issue regression when using a HSM to store TLS private keys.
+* Previously, a rare error scenario could occur where a node would erroneously perceive a valid connection to a peer when, in fact, it was not connected. This issue typically arose when the peer node was disconnecting/connecting. This issue has now been resolved.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.10/enterprise/release-notes-enterprise.md:36]:
```suggestion
* Previously, a rare error scenario could occur where a node would erroneously perceive a valid connection to a peer when, in fact, it was not connected. This issue typically arose when the peer node was disconnecting/connecting. 
```

File: content/en/platform/corda/4.11/community/release-notes.md [hunk 5]
```diff
@@ -20,6 +20,20 @@ tags:
...
 # Corda Community Edition 4.11 release notes
 
+## Corda Community Edition 4.11.2 release notes
+
+Corda Community Edition 4.11.2 is a patch release of Corda Community Edition focused on resolving issues.
+
+### Fixed issues
+
+* In the default log4j2.xml file, the Delete action in the DefaultRolloverStrategy policy for log files beginning with `diagnostic-*` or `checkpoints_agent-*`  was incorrect. It erroneously compared against the wrong file names. This issue has been rectified, ensuring that files are now deleted in accordance with the policy.
+* Previously, a rare error scenario could occur where a node would erroneously perceive a valid connection to a peer when, in fact, it was not connected. This issue typically arose when the peer node was disconnecting/connecting. This issue has now been resolved.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.11/community/release-notes.md:30]:
```suggestion
* Previously, a rare error scenario could occur where a node would erroneously perceive a valid connection to a peer when, in fact, it was not connected. This issue typically arose when the peer node was disconnecting/connecting.
```

File: content/en/platform/corda/4.11/community/release-notes.md [hunk 6]
```diff
@@ -20,6 +20,20 @@ tags:
 
 # Corda Community Edition 4.11 release notes
 
+## Corda Community Edition 4.11.2 release notes
+
+Corda Community Edition 4.11.2 is a patch release of Corda Community Edition focused on resolving issues.
+
+### Fixed issues
+
+* In the default log4j2.xml file, the Delete action in the DefaultRolloverStrategy policy for log files beginning with `diagnostic-*` or `checkpoints_agent-*`  was incorrect. It erroneously compared against the wrong file names. This issue has been rectified, ensuring that files are now deleted in accordance with the policy.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.11/community/release-notes.md:29]:
```suggestion
* In the default log4j2.xml file, the Delete action in the DefaultRolloverStrategy policy for log files beginning with `diagnostic-*` or `checkpoints_agent-*` was incorrect. It erroneously compared against the wrong file names. This issue has been rectified, ensuring that files are now deleted in accordance with the policy.
```

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md [hunk 7]
```diff
@@ -17,6 +17,31 @@ weight: 10
...
+
+### Fixed issues
+
+* In the default log4j2.xml file, the Delete action in the DefaultRolloverStrategy policy for log files beginning with `diagnostic-*` or `checkpoints_agent-*`  was incorrect. It erroneously compared against the wrong file names. This issue has been rectified, ensuring that files are now deleted in accordance with the policy.
+* Resolved a TLS connection issue regression when using a HSM to store TLS private keys.
+* Fixed a regression in the RPC `getFlowsMatchingV2` extension operation that rendered it incompatible with previous Corda releases. This issue manifested in the Corda flow management GUI, preventing it from displaying the flow status of previous Corda releases. Due to this fix, if you have created an RPC client using 4.11.1 or 4.11, you need to rebuild the client using 4.11.2.
+* Previously, a rare error scenario could occur where a node would erroneously perceive a valid connection to a peer when, in fact, it was not connected. This issue typically arose when the peer node was disconnecting/connecting. This issue has now been resolved.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:37]:
```suggestion
* Previously, a rare error scenario could occur where a node would erroneously perceive a valid connection to a peer when, in fact, it was not connected. This issue typically arose when the peer node was disconnecting/connecting. 
```

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md [hunk 8]
```diff
@@ -17,6 +17,31 @@ weight: 10
...
+
+Corda Enterprise Edition 4.11.2 is a patch release of Corda Enterprise Edition focused on resolving issues.
+
+### Upgrade recommendation
+
+As a developer or node operator, you should upgrade to the [latest released version of Corda]({{< relref "../enterprise/_index.md" >}}) as soon as possible. The latest Corda Enterprise release notes are on this page, and for the latest upgrade guide, refer to [Upgrading a CorDapp or node](upgrading-index.md).
+
+### Fixed issues
+
+* In the default log4j2.xml file, the Delete action in the DefaultRolloverStrategy policy for log files beginning with `diagnostic-*` or `checkpoints_agent-*`  was incorrect. It erroneously compared against the wrong file names. This issue has been rectified, ensuring that files are now deleted in accordance with the policy.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:34]:
```suggestion
* In the default log4j2.xml file, the Delete action in the DefaultRolloverStrategy policy for log files beginning with `diagnostic-*` or `checkpoints_agent-*` was incorrect. It erroneously compared against the wrong file names. This issue has been rectified, ensuring that files are now deleted in accordance with the policy.
```

File: content/en/platform/corda/4.10/community/release-notes.md (hunk 1 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.10/community/release-notes.md:30]: Updated

File: content/en/platform/corda/4.10/enterprise/release-checksum-enterprise.md (hunk 2 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.10/enterprise/release-checksum-enterprise.md:31]: Updated

File: content/en/platform/corda/4.10/enterprise/release-notes-enterprise.md (hunk 4 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.10/enterprise/release-notes-enterprise.md:36]: Updated

File: content/en/platform/corda/4.11/community/release-notes.md (hunk 6 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.11/community/release-notes.md:29]: Updated

File: content/en/platform/corda/4.11/community/release-notes.md (hunk 5 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.11/community/release-notes.md:30]: Updated

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md (hunk 7 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:37]: Updated

**Review by nadinequinr3 — APPROVED** [review]

**Pull request merged**